crates/core/tedge_mapper/src/c8y/converter.rs

```diff
@@ -354,6 +358,18 @@ where
         self.alarm_converter = AlarmConverter::Synced;
         sync_messages
     }
+
+    fn process_operation_update_messages(&mut self, message: &Message) {
```

Comment:
This function is doing half of its job. No only the internal list of operations must be updated but Cumulocity must be informed. This function must return a SmartRest message of type 114 to be sent to Cumulocity.

This signature of this function must be similar to `try_init_messages` and should invoke the `create_supported_operations_fragments_message` function,